spec/app/domain/authentication/authn-jwt/restriction_validation/validate_restrictions_included_spec.rb

```diff
@@ -0,0 +1,140 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+
+RSpec.describe('Authentication::AuthnJwt::RestrictionValidation::ValidateRestrictionsIncluded') do
+  let(:decoded_token) {
+    {
+      "claim1" => %w[a b],
```

Comment:
would you like to test also non string array members like `["string", 1, "another string", 100]`
and object members may be too